examples/gallery/embellishments/timestamp.py

```diff
@@ -0,0 +1,18 @@
+"""
+Timestamp
+---------
+The :meth:`pygmt.Figure.timestamp` method can draw the
+GMT timestamp logo on the map/plot. A custom label
+can be added via the ``label`` parameter.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
```

Comment:
It's better to also plot a map via `basemap` or `coast`, so that users know that the timestamp will be shown on the bottom-left corner of their figures.